### PR TITLE
Issue 5958. Provide method to get the pager's element ID

### LINKED ISSUE #5958: Provide a method to get the pager's element ID.
## Description of the need

This is a cross-port from https://www.drupal.org/node/702778, and part of the task list #5770. The D7 issue added a method, `getElement()`, to `PagerDefault` that returns the element ID of the pager. Having this method allows developers an easier way to have more than one pager in the same page without conflicts. 

Before this new method, the only way to handle more than one pager in the same page was to hard-code the element ID through `PagerDefault::element()`. This doesn't make much sense and can lead to unexpected conflicts if two different modules need to add a pager to the same page. `PagerDefault` already has a way to avoid those conflicts by keeping track of IDs in a static variable, but until this feature was added in D7, it was impossible to get the internally assigned element ID (which is a protected property). 

PR coming.

### PR DESCRIPTION
**Pull request by argiepiano**
Fixes https://github.com/backdrop/backdrop-issues/issues/5958

### CODE REVIEW

File: core/includes/pager.class.inc [hunk 1]
```diff
@@ -168,4 +168,19 @@ class PagerDefault extends SelectQueryExtender {
...
     return $this;
   }
+
+  /**
+   * Gets the element ID for this pager query.
+   *
+   * The element is used to differentiate different pager queries on the same
+   * page so that they may be operated independently.
+   *
+   * @return
```
Comment (klonos) [inline core/includes/pager.class.inc]: This should stop PHPCS from complaining:
```suggestion
   * @return int
```

**Review by klonos — APPROVED** [review]
LGTM now 👍🏼

File: core/includes/pager.class.inc [hunk 2]
```diff
@@ -168,4 +168,19 @@ class PagerDefault extends SelectQueryExtender {
...
+  /**
+   * Gets the element ID for this pager query.
+   *
+   * The element is used to differentiate different pager queries on the same
+   * page so that they may be operated independently.
+   *
+   * @return int
+   *   Element ID that is used to differentiate between different pager
+   *   queries.
+   */
```
Comment (quicksketch) [inline core/includes/pager.class.inc]:
```suggestion
   *   queries.
   *
   * @since 1.24.1 Method added.
   */
```

**Comment by quicksketch** [comment]
Looks good to me! I added a `@since` docblock line to indicate this will be in 1.24.1.

**Comment by quicksketch** [comment]
Merged into 1.x and 1.24.x. Thanks!

**Pull request merged**